README.md

```diff
@@ -154,98 +116,109 @@ type: tutorial
 
 ### Step 2: Install the CrunchyDB Operator
 
-- Navigate to **OpenShift web console** which was launched during previous step. Select **operatorhub** under Operators and type 'Crunchy' in the search bar.
+1. Navigate to the **OpenShift web console** which you launched in previous step. Select **OperatorHub** under Operators and type "Crunchy" in the search bar.
+
+    ![](doc/source/images/install-operator1.png)
+
+1. Select **Crunchy Postgres Operator** (non custom) and click **Install**.
+
+    ![](doc/source/images/install.png)
+
+1. Create an Operator Subscription by choosing All namespaces or specific namespace (select default project crunchy-project) and click **Subscribe**.
+
+    ![](doc/source/images/subscribe1.png)
 
-![](doc/source/images/install-operator1.png)
+1. After a few minutes, the operator is installed on the cluster. Verify your installation by selecting **Installed Operators** under `Operators` and make sure the status shows as Succeeded.
 
-- Click on **Crunchy Postgres Operator** (non custom) and click **install**.
+    ![](doc/source/images/installed-operator.png)
 
-![](doc/source/images/install.png)
+### Step 3: Connect to the Openshift Cluster in your command line interface
 
-- Create Operator Subscription by choosing All namespaces or specific namespace (select default project crunchy-project) and click **subscribe**.
+1. Log in to the Red Hat OpenShift on IBM Cloud Kubernetes Service cluster through your CLI. To log in, you need a token which can be genrated after you log into the Openshift Cluster web console. See below screenshot to `copy the path`.<!--EM: What does "copy the path" mean?-->
 
-![](doc/source/images/subscribe1.png)
+    ![](doc/source/images/Login-CopyCommand.png)
 
-- After a couple of minutes, the operator gets installed on the cluster. We can verify by clicking on Installed Operators under `Operators` and see that the operator is successfully installed with status showing as Succeeded.
+1.  new window will open requesting the login token details. Copy the login token. 
+    ![](doc/source/images/Login-Token.png)
 
-![](doc/source/images/installed-operator.png)
+1. In your terminal, paste the login token. Once you log in, you should see a screen that has information about your login name, your server, the token, and the projects.
 
-### Step 3: Connect to the Openshift Cluster in CLI (Command Line Interface)
+    ![](doc/source/images/CLI-Login.png)
 
-- Login to the ROKS(IBM Managed) Openshift cluster through CLI(command line Iterface). 
-To login you would require token which can be genrated after you login to Openshift Cluster web console. See below screenshot to `copy the path`.
-![](doc/source/images/Login-CopyCommand.png)
+### Step 4: Create and deploy a CrunchyDB Operator on an OpenShift Cluster and create a database
+<!--EM: In this section, what are `pgo` and pvc`-->
```

Comment:
These are libraries that are used in terminal to Connect Connect to the Openshift Cluster